internal/pkg/ai-functions/handler/handler_ai_function.go

```diff
@@ -139,6 +92,96 @@ func (h *AIFunction) Apply(ctx context.Context, req *pb.ApplyRequest) (pbValue *
 	return v, nil
 }
 
+func (h *AIFunction) createTestCaseForRequirementIDAndTestID(ctx context.Context, factory functions.FunctionFactory, req *pb.ApplyRequest, openaiURL *url.URL) (any, error) {
```

Comment:
move test-case related functions to another file, maybe `handler_ai_function_test_case.go`.

Reply:
Ok